paddle/fluid/framework/cryption_test.cc

```diff
@@ -0,0 +1,284 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "glog/logging.h"
+#include "gtest/gtest.h"
+#include "paddle/fluid/framework/cryption.h"
+
+#define TEST_BUF_SIZE 32
+
+using paddle::framework::Cryption;
+
+void readFile(const std::string& path, char* buf, size_t len) {
+  std::ifstream fin;
+
+  fin.open(path, std::ios::binary);
+  fin.read(buf, len);
+
+  buf[len] = '\0';
+
+  fin.close();
+  return;
+}
+
+void writeFile(const std::string& path, const char* buf, size_t len) {
+  std::ofstream fout;
+
+  fout.open(path, std::ios::binary);
+  fout.write(buf, len);
+
+  fout.close();
+  return;
+}
+
+TEST(Cryption, OnceInitTest) {
+  Cryption& c1 = *Cryption::GetCryptorInstance();
```

Comment:
what's the purpose of this unittest, c1 & c2 looks like the same pointer?

Reply:
Done, deleted